cli/kurtosis/backend/service_network.go

```diff
@@ -173,7 +177,9 @@ func (network *KurtosisTestServiceNetwork) UpdateFilesArtifact(fileArtifactUuid
 }
 
 func (network *KurtosisTestServiceNetwork) GetUniqueNameForFileArtifact() (string, error) {
-	return "", unimplemented("GetUniqueNameForFileArtifact")
+	network.fileArtifactNameCounter++
```

Comment:
wouldn't it be clearer to use sync/atomic ?
Then you could just drop the test (which only passes because there's no concurrency)

Reply:
Oh the test tests that the names are unique, it's not concerned with concurrency (so regardless of the implementation the test should stay, I can make it run in parallel though)